scripts/runalltests.sh

```diff
@@ -0,0 +1,37 @@
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/testslog
+testdir=$workdir/../../../tests
+
+if [ -d $workdir ]; then
```

Comment:
originally it was intended to store multiple log files and an archive of this files